proposed/apphost-embed-install-location.md

```diff
@@ -104,6 +104,13 @@ explicitly set to a value that does not include `AppRelative`, then setting
 `AppHostRelativeDotNet` is meaningless - the SDK will not write the relative
 path into the `apphost` and the `apphost` will not check for a relative path.
 
+The path must not be rooted and will be considered relative to the app location
+(the location of the `apphost` itself) at run time. For example, when running
+`C:\dir\app.exe` or `/home/dir/app` with `AppHostRelativeDotNet` set to:
+  - `my_dotnet`: the app will look at `C:\dir\my_dotnet` or `/home/dir/my_dotnet`
+  - `./my_dotnet`: the app will look at `C:\dir\my_dotnet` or `/home/dir/my_dotnet`
+  - `../my_dotnet` the app will look at `C:\my_dotnet` or `/home/my_dotnet`
```

Comment:
Thanks! It would be good to know what the behavior would be if `../../` ends up going past drive root. Is there a check for that case or its undefined?

Reply:
I think the meta point here is whatever the user defines as the relative path it is unmodified defers to the under lying OS file system APIs. I wouldn't do much more than provide an example of how we use it and state the precise semantics are at the mercy of the underlying file system.